### DISCUSSION TITLE
Lesson text a little too long in some places

### DISCUSSION THREAD

**Discussion by maranr** [GDQuest/learn-gdscript#152] [Ideas]
I would break up some of the lessons into subpages to avoid scrolling. I found that some sections, like the one on variables, a little long with a number of topics on one page that could be broken up more to keep focus.

**Comment by NathanLovato** [comment]
Definitely. We're aware of that and plan at least to shorten new lessons (and split them up more). We should then revisit older lessons and do the same where possible.
